region/maridia/outer/Fish Tank.json

```
@@ -3568,6 +3568,47 @@
       "requires": [],
       "flashSuitChecked": true
     },
+    {
+      "link": [6, 2],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        "Gravity",
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"or": [
+              {"partialRefill": {"type": "ReserveEnergy", "limit": 5}},
+              {"and": [
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+                {"or": [
+                  "canBePatient",
+                  "Plasma",
+                  "ScrewAttack",
+                  {"and": [
+                    "canWalljump",
+                    {"canShineCharge": {"usedTiles": 33, "openEnd": 0}},
+                    {"shinespark": {"frames": 2, "excessFrames": 2}}
+                  ]}
+                ]}
+              ]}
+            ]}
```

with full missiles and killing the 2 available skulteras, you have almost a 90% chance of gaining some reserve energy (and you could likely farm the other 2 with a gravity jump or wall jump). After the strat, you're blue and can farm the 4 pirates. Maybe this isnt the best way to be representing these, but I don't think this strat needs all these extra options
```suggestion
            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
```